drivers/input/i8042prt/pnp.c

```diff
@@ -3,7 +3,7 @@
  * LICENSE:     GPL - See COPYING in the top level directory
  * FILE:        drivers/input/i8042prt/pnp.c
  * PURPOSE:     IRP_MJ_PNP operations
- * PROGRAMMERS: Copyright 2006-2007 Herv� Poussineau (hpoussin@reactos.org)
```

Comment:
This change doesn't belong here.

Reply:
idk why this happened.

Reply:
I GET IT! IT'S HAPPENED DUE TO GITHUB CONVERTED ENCODING! FROM ISO-** TO UTF-8!